From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Wed, 28 Dec 2022 16:40:57 +0100
Subject: metainfo: Drop Purism::form_factor

The duplicate key (rightfully) tips up metainfo validation
and we also have `display_length` set as well.
---
 data/org.sigxcpu.MobileSettings.metainfo.xml.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/data/org.sigxcpu.MobileSettings.metainfo.xml.in b/data/org.sigxcpu.MobileSettings.metainfo.xml.in
index c026ad3..15d1cda 100644
--- a/data/org.sigxcpu.MobileSettings.metainfo.xml.in
+++ b/data/org.sigxcpu.MobileSettings.metainfo.xml.in
@@ -23,11 +23,6 @@
     <control>touch</control>
   </recommends>
 
-  <custom>
-    <value key="Purism::form_factor">workstation</value>
-    <value key="Purism::form_factor">mobile</value>
-  </custom>
-
   <url type="homepage">https://gitlab.gnome.org//guidog/phosh-mobile-settings</url>
   <url type="bugtracker">https://gitlab.gnome.org//guidog/phosh-mobile-settings/issues</url>
 
